plus/pom.xml

```diff
@@ -55,6 +55,14 @@ limitations under the License.
       <groupId>org.apache.calcite.avatica</groupId>
       <artifactId>avatica-core</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.calcite.avatica</groupId>
+      <artifactId>avatica-server</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.eclipse.jetty</groupId>
+      <artifactId>jetty-server</artifactId>
```

Comment:
I'm not seeing any explicit additions of jetty-server classes to `plus/src` as a part of this changeset. Why did you add `jetty-server` here?

Reply:
This is for satisfying the maven checks. Avatica server indirectly imports this, so the checker fails with indirect dependency.

Reply:
The point of transitive dependencies is that they come in automatically. You should not add them as a dependency unless you are using classes from them. What is the error you're seeing without this?